CoreFoundation/Base.subproj/CFInternal.h

```diff
@@ -177,6 +177,12 @@ CF_PRIVATE CFIndex __CFActiveProcessorCount();
     #else
         #error Compiler not supported
     #endif
+#elif defined(__s390x__)
+    #if defined(__GNUC__)
+        #define HALT do {__builtin_trap(); kill(getpid(), 9); __builtin_unreachable(); } while (0)
+    #else
+        #error Compiler not supported
+    #endif
```

Comment:
BTW, it would be nice to rebase this on https://github.com/apple/swift-corelibs-foundation/pull/372 .  It collapses the duplication.